packages/components/fields/search-select-field/src/search-select-field.js

```diff
@@ -0,0 +1,278 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import requiredIf from 'react-required-if';
+import {
+  SafeHTMLElement,
+  filterDataAttributes,
+} from '@commercetools-uikit/utils';
+import Constraints from '@commercetools-uikit/constraints';
+import Spacings from '@commercetools-uikit/spacings';
+import FieldLabel from '@commercetools-uikit/field-label';
+import SearchSelectInput from '@commercetools-uikit/search-select-input';
+import FieldErrors from '@commercetools-uikit/field-errors';
+
+const hasErrors = (errors) => errors && Object.values(errors).some(Boolean);
+const searchSelectInputProps = Object.keys(SearchSelectInput.propTypes);
+const filterSerachSelectFieldProps = (props) => {
```

Comment:
Just a helper function to filter out wrong (invalid) props. It checks if the prop is allowed by `SearchSelectInput` and if yes, then adds it to the `filteredProps` list (valid list of props).

Reply:
Could you add this as a comment on the code? 🙂